crt_portal/cts_forms/templates/forms/base.html

```diff
@@ -53,19 +53,25 @@
       </div>
     </div>
 
-    <main>
+    <header class="page-header-background">
       <div class="grid-container">
         <div class="grid-row">
           <div class="grid-col-1"></div>
           <div class="grid-col-10">
             <h1>
               Contact the Department of Justice about a civil rights concern
             </h1>
-            {% block content %} {% endblock %}
+            <p class="intro-subtitle">
```

Comment:
This is the template base we will want to use for all or many of the pages. So we will want to add a block here- something like `{% block intro %}{%  endblock %}` and then we can add the text on the form page in a corresponding block.

Reply:
Hmmmm, from what I see of the designs, every single page has the exact same intro text. We could extract this out into a block, but I think that would be a premature optimization.

Reply:
We know we will need a homepage and, generally, we want to make software that is easy to extend. I would agree with you about it being premature if it was more work to implement in a way that made it extensible, but since it is a wash, let's get that in now.

Reply:
OK, sounds good. Done in e78fbecfc4a1dfab94a4a6e60b0d8b1c0fd53657. I extracted out the whole header area so that the h1 title, intro subtitle, and other details of the header can be customized from page to page.